src/textpath.js

```diff
@@ -28,14 +28,22 @@ SVG.TextPath = SVG.invent({
 
       return this
     }
+    // return the array of the path track element
+  , array: function() {
+      var track = this.track()
+
+      return (track) ? track.array() : null
```

Comment:
I don't know why I put parens :laughing: on this one I will remove them. I won't be able to use the boolean operator here unfortunately as I have to return the array of the track and not just the track.